src/containers/MainTabsNavigator/Containers/NotificationsManager/components/NotificationCard.tsx

```diff
@@ -10,16 +21,43 @@ import {
   NotificationCardDetails,
   NotificationCardRow,
   NotificationSwitchContainer,
+  DeleteButton,
+  DeleteButtonText,
+  DeleteButtonContainer,
 } from './NotificationCard.style';
 
-const BUTTON_OPACITY_LEVEL = 0.6;
+const DISTANCE_FROM_END = 50;
+const DELETE_BUTTON_WIDTH = 80;
+
+type ContextType = {
+  translateX: number;
+  translateY: number;
+};
 
 export const NotificationsCard: FC<{notification: NotificationResponse}> = ({
   notification,
 }) => {
+  const swipeAnimationValue = useSharedValue(0);
```

Comment:
Would recommend a rename to imply the intended use for this const:
ex1:
```js
  const translateXAnimationValue = useSharedValue(0);
```
ex2:
```js
  const deleteButtonXAnimatedValue = useSharedValue(0);
```